.gitignore

```diff
@@ -16,3 +16,4 @@ docs/inventory/
 .coverage.*
 .venv/
 coverage.xml
+*_out/
```

Comment:
I would say "examples/*_out". And don't forget the final newline at the end of the file :)